### ISSUE TITLE
Invisible trollager blocks and possible vanilla mobs issue

### ISSUE THREAD

**Issue by Lictor000** [Daveyx0/PrimitiveMobs#95]
Hi, I installed this really cool mod and started roaming around to meet the new mobs. At one time I met a trollager that started throwing blocks at me, but sometimes some of them were invisible.

Also after a couple of minecraft nights I felt that something was missing. Then I realized I didn't met a single zombie or regular skeleton after two full nights, and the mobs number was very low (met some single creeper or regular spider, but no groups of them).
Did I miss something from the config options or is this intended?

**Comment by Daveyx0** [comment]
Thanks for the report. Working on that invisibility thing. I dont think my mod changes mob spawns, could be another mod causing it?

**Comment by Lictor000** [comment]
Hi, it was a couple of weeks ago, since I had another couple of mods that interact with mobs I tought of that, but I'm pretty sure I tried using only your mod to be sure of it before reporting the isssue.
I remember even trying some older versions of the mod before the 1.1.6 beta.
Anyway I will gladly try to replicate the issue this evening when I get home (Europe time here).

**Comment by Lictor000** [comment]
Ok here I am with some additional info. I created, via multiMC, a minecraft instance with just Primitive Mobs 1.1.6 beta and the last recommended Forge version.
I created a superflat world to better study the mob diversity at night. And then something weird that I hope can give some help happened.
The moment I run the command to set the time to night (/time set 18000), mobs were generated around me, and everything looked like as I think like it should be:
![2018-06-19_19 33 29](https://user-images.githubusercontent.com/39945124/41614571-268f9ad0-73f9-11e8-8f33-a7a235524455.png)
![2018-06-19_19 33 32](https://user-images.githubusercontent.com/39945124/41614572-26d62946-73f9-11e8-8c78-ad55a363be19.png)
![2018-06-19_19 33 24](https://user-images.githubusercontent.com/39945124/41614574-26f5b216-73f9-11e8-93b4-b1d1c65e0546.png)
![2018-06-19_19 33 27](https://user-images.githubusercontent.com/39945124/41614576-271508b4-73f9-11e8-8889-6049b4e12835.png)
As you can see there's a fairly ammount of both vanilla mobs and primitive mobs looking in the four directions around me.
Then I started moving around flying through creative mode and, gotten far from the point of previous screenshots, the the situation was more like this:
![2018-06-19_19 34 10](https://user-images.githubusercontent.com/39945124/41614677-60530770-73f9-11e8-8f0a-61ee84b761c0.png)
![2018-06-19_19 34 21](https://user-images.githubusercontent.com/39945124/41614678-6072680e-73f9-11e8-9182-5c43db824ddd.png)
![2018-06-19_19 34 25](https://user-images.githubusercontent.com/39945124/41614679-609b611e-73f9-11e8-9fb7-3a0823f3efb8.png)
![2018-06-19_19 34 36](https://user-images.githubusercontent.com/39945124/41614680-60b8efcc-73f9-11e8-8e18-3bbab4563741.png)
![2018-06-19_19 35 07](https://user-images.githubusercontent.com/39945124/41614681-60d5581a-73f9-11e8-932f-4bc85c613337.png)
![2018-06-19_19 35 19](https://user-images.githubusercontent.com/39945124/41614682-60fc3b74-73f9-11e8-8188-15936062196e.png)

Sure, you can see a vanilla creeper and a group of vanilla skeletons in these screenshots, but finding them was like finding a needle in a haystack. As I mentioned in my first post in a couple of minecraft days in my survival try of the mod I didn't hear even a single zombie growl.

So then I tried modifing spawn rates via config setting them really low for every mob (dividing everything by 10) and this sort of "fixed" the problem.

I'm no coder, but it seems that sometimes (like when I set night time) mobs gets generated in a 50-50 rate between vanilla and modded, and most of the time prefer the modded ones over the vanilla ones. Statistically, every now and then I faced an area better populated with vanilla mobs, but they were sporadic cases.

What are the numbers in the config options? Percentages? Maybe setting a probability to check if the spawned mob will be a vanilla or primitive one and then select which primitive will be based on config rates? Configurable maybe?

**Comment by Lictor000** [comment]
By curiosity, since I just discovered this cool mod and its awesome mobs (they're definetly my favourites among every other modded mobs), I tried the mod version for Minecraft 1.7.10. By comparison of mob population between old a new mod version I can confirm that there's something that either interferes with vanilla mobs generation, making their effective spawn rate very low, or with default spawn rate values of modded mobs, making them really high. I didn't take screenshots this time, but while in 1.7.10 version there was a nice equilibrium with vanilla mobs in the new one at one point a found myself literally surrounded by spider families.
Comparison again made installing only Primitive mobs on a fresh minecraft install.
